python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/dhcp-servers.j2

```diff
@@ -65,6 +77,18 @@
 {%                     for subnet in dhcp_server_subnets %}
 | {{ subnet.subnet }} | {{ subnet.name | arista.avd.default("-") }} | {{ subnet.dns_servers }} | {{ subnet.default_gateway }} | {{ subnet.lease_time }} | {{ subnet.ranges }} |
 {%                     endfor %}
+{%                     for subnet in dhcp_server.subnets | arista.avd.natural_sort %}
+{%                         if subnet.reservations is arista.avd.defined %}
```

Comment:
It will be better to add a title `DHCP reservations` at Subnets level and then add `DHCP reservations in {{ subnet.subnet }}` titles under `DHCP reservations`.

Reply:
I think having the sub-heading suggested in the comment above is fine.